doc/running_on_phone.md

```diff
@@ -35,21 +38,47 @@ Files required to boot from eMMC are:
 
 ### Uploading it to the eMMC
 
+####  via bootloader
+
+On linux:
 -  Use the `pureflash` command to burn image into the eMMC card `sudo pureflash <PurePhone.img> /dev/sdX` **Note:** Replace `/dev/sdX` with the disk name you found for your phone after connecting it to your computer.
 
+On MacOS you can use script:
+- `./tools/macflash.sh`
+
+**NOTE:** Devices in the field have direct eMMC via bootloader locked.
+
+#### via uptader utility
+
+Every device is loaded with additional update software which is capable of updating any software package onto the device, while keeping last software backup.
+Updater utility is the only normal way to update devices running in field and is used by MuditaCenter.
+
+**NOTE:** Porviding a minimum viable update package and utility for Open Source community is on our backlog. It can be created by trimming additional developemnt api from:
+[updater feature tests](https://github.com/mudita/QAMuditaOS/tree/master/scenarios/updater). You can ask for development version of MuditaCentre with "update from file" capability enabled.
+
+Update packede is defined here: [build targets](./build_targets.md)
+
 ## How to mount Mudita Pure as a USB MSC
 
-There are two ways to mount the phone as a USB MSC - using a bootloader or Ozone debugger.
+**NOTE:** all phones in field have eMMC access locked with password, following documentation is only for development devices or ones with eMMC unlocked or bypassed.
+
+There are two ways to mount the phone as a USB MSC to access full access to the memory - using a bootloader or Ozone debugger.
 
 ### Mounting using a bootloader
 
-If you have a bootloader flashed and working use the following key combination, to mount Mudita Pure as MSC:
+If you have a bootloader flashed with unlocked eMMC access use the following key combination, to mount Mudita Pure as MSC:
 
-- hold centre button (selection)
-- briefly press reset button (the one closest to the phone on the green breakout board)
-- wait while text scrolls from the bottom
+- hold center button (selection)
+- reset the device, either:
+    - briefly press reset button (the one closest to the phone on the green breakout board)
+    - detach USB and battery, then attach battery then USB
+- wait while text scrolls from the bottom - still holding the center button
 - release centre button
 - press key 4
```

Comment:
Isn't key 9 used right now?

Reply:
This depends - I actually didn't change the documentation here except adding information that it's for unprotected bootloader :)